thunder/__init__.py

```diff
@@ -1014,6 +1014,16 @@ def last_compile_options(fn: Callable, /) -> None:
         print(f"\t{option}")
 
 
+def print_auto_registered_torch_ops(fn: Callable, /) -> set[str] | None:
+    """Returns a set of auto-registered Torch operator names present in the given JIT-compiled function."""
+    op_names = set()
+    trc = last_traces(fn)[0]
+    for bsym in trc.bound_symbols:
+        if (meta_func_name := getattr(bsym.sym.meta, "__name__", None)) and meta_func_name == "meta_func":
```

Comment:
Looks great overall, thank you for revising.
Nit: Could this be
```suggestion
        if getattr(bsym.sym.meta, "__name__", None) == "meta_func":
```
or even better bsym.sym.meta directly to the meta func?

Reply:
`bsym.sym.meta` is the inner function returned by `thunder.torch.meta_adaptor`, I haven't found other checks can directly compare `bsym.sym.meta` to `thunder.torch.meta_adaptor.<locals>.meta_func`, probably checking the names is sufficient? @t-vi